devbuild.sh

```diff
@@ -161,7 +161,7 @@ if [ "${VERBOSE}" = true ] ; then
 fi
 
 # set ENV_FILE for this platform/compiler combination
-ENV_FILE="${SRC_DIR}/env/build_${PLATFORM}_${COMPILER}.env"
+ENV_FILE="${SRC_DIR}/modulefiles/build_${PLATFORM}_${COMPILER}.env"
```

Comment:
@christopherwharrop-noaa, I agree. This will be my next PR. I don't want to put several topics into one PR. I'll modify the env files into the regular module files.

Reply:
@christopherwharrop-noaa, if you want this change in this PR, I'll do it soon.